website/versioned_docs/version-0.12.0/quick-start-guide.md

```diff
@@ -14,7 +14,8 @@ data both snapshot and incrementally.
 
 ## Setup
 
-Hudi works with Spark-2.4.3+ & Spark 3.x versions. You can follow instructions [here](https://spark.apache.org/downloads) for setting up Spark.
+Hudi works with Spark-2.4.3+ & Spark 3.x versions. You can follow
```

Comment:
I am assuming this has same exact changes as previous file. please address whatever comments given above to this file as well